public/svgs/app/user.svg

```diff
@@ -0,0 +1,74 @@
+<svg viewBox="0 0 35 35">
+	<g>
+		<path style="fill:#E8E8E8;" d="M17.462,6.65H17.4l-0.005,0c-3.689,0.038-6.658,3.058-6.658,6.767c0,0.562,0.067,1.107,0.196,1.628

```

Comment:
I'm sure there are things like that - we'll need to hook up a build process for the svgs - clean, lint, optimize, package